pxb/v2/jenkins/percona-xtrabackup-2.4-test-param.yml

```diff
@@ -11,9 +11,9 @@
     properties:
     - build-discarder:
         days-to-keep: -1
-        num-to-keep: 50
+        num-to-keep: 30
```

Comment:
Is this the number of test params to keep or the number of test pipelines?

We have 8 OS that get tested against 4 different server, resulting in 32 test pipelines for each test param

Reply:
This is only for param runs, not configurations and not of pipeline

Reply:
ack